partrt/partrt

```diff
@@ -266,9 +266,9 @@ get_monotonic()
 # $1 File
 # $2 Value
 write_to_file () {
-    local -ri start=$(get_monotonic)
-    local -r file="$1"
```

Comment:
Checking documentation for Bash 5.1, ’local’ takes same arguments as ’declare’, and ’declare’ accepts both ’-i’ and ’-r’. So I'm confused as to why this line gives you that error.